sharpd/sharp_vty.c

```diff
@@ -185,12 +185,27 @@ DEFPY (remove_routes,
 	return CMD_SUCCESS;
 }
 
+DEFUN_NOSH (show_debugging_sharpd,
+	    show_debugging_sharpd_cmd,
```

Comment:
since we're doing space alignment here anyway the macro arguments should probably line up with the first one

Reply:
They are?  What am I missing?

Reply:
Disregard me, github shows tabs as 6 columns